src/Grpc/test/InteropTests/Helpers/WebsiteProcess.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Internal;
+using Xunit.Abstractions;
+
+namespace InteropTests.Helpers
+{
+    public class WebsiteProcess : IDisposable
+    {
+        private readonly Process _process;
+        private readonly ProcessEx _processEx;
+        private readonly TaskCompletionSource<object> _startTcs;
+        private readonly StringBuilder _consoleOut = new StringBuilder();
+        private readonly string _serverLogPath;
+        private static readonly Regex NowListeningRegex = new Regex(@"^\s*Now listening on: .*:(?<port>\d*)$");
+
+        public string ServerPort { get; private set; }
+
+        public WebsiteProcess(string path, ITestOutputHelper output)
+        {
+            var attributes = Assembly.GetExecutingAssembly()
+                .GetCustomAttributes<AssemblyMetadataAttribute>();
```

Comment:
Oh, but we should revisit things like this, burning paths into the assembly doesn't work on helix, can we have this fallback to a relative path default that just works (that would be the helix path)

Reply:
Like if the attribute doesn't exist, just fallback to "/artifacts/log" which is the default place we upload logs from on helix

Reply:
The problem here isn't just about the path. Specifically, in this case, the test executable is trying to run the server/client executables but I don't think they are included in the Helix test payload. In general, we need a pattern for including other executables in test payloads.

However, in this particular case, I believe it might be possible to run the tests in process, in which case, we should be able to avoid the problem all together.